src/flashsystem-dashboard.tsx

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright contributors to the ibm-storage-odf-console project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+ import * as React from 'react';
+ import * as _ from 'lodash';
+ import { Helmet } from "react-helmet";
+ import { HorizontalNav, PageHeading } from "@console/dynamic-plugin-sdk/provisional";
+ import { Grid, GridItem } from "@patternfly/react-core";
+ 
+ import StorageEfficiencyCard from './components/storage-efficiency-card/storage-efficiency-card';
+
+ const UpperSection: React.FC = (props) => {
+   return (
+     <Grid hasGutter>
+       <GridItem span={3}>          
+          <StorageEfficiencyCard/>
+       </GridItem>
+       <GridItem span={6}>         
+       </GridItem>
+       <GridItem span={3} >
+       </GridItem>
+     </Grid>
+   );
+ };
+ 
+ const FlashsystemDashboard: React.FC = (props) => {
+   return (
+     <>
+       <div className="co-dashboard-body">
+         <UpperSection {...props}/>
+       </div>
+     </>
+   );
+ };
+ 
+ const FlashsystemDashboardPage: React.FC<any> = (props) => {
```

Comment:
You can define types explixtly here for `match`. I see thats only sued below.

Reply:
compile error. 
`TS2741: Property 'match' is missing in type '{ children?: ReactNode; }' but required in type '{ match: match<{ namespace: string; name: string; }>; }'.`